Pinta.Effects/Effects/ReliefEffect.cs

```diff
@@ -38,38 +38,38 @@ public override void LaunchConfiguration ()
 	#region Algorithm Code Ported From PDN
 	public override void Render (Cairo.ImageSurface src, Cairo.ImageSurface dst, Core.RectangleI[] rois)
 	{
-		base.RenderColorDifferenceEffect (Weights, src, dst, rois);
+		var weights = Weights ();
+		base.RenderColorDifferenceEffect (weights, src, dst, rois);
 	}
 
-	private double[][] Weights {
-		get {
-			// adjust and convert angle to radians
-			double r = (double) Data.Angle * 2.0 * Math.PI / 360.0;
+	private double[][] Weights ()
```

Comment:
Maybe a name like `BuildWeights()` or `ComputeWeights()` for the method name?

Reply:
No problem, I just did it